### ISSUE TITLE
[Feature Request] Less Verbose Encounter Difficulty

### ISSUE THREAD

**Issue by SlRvb** [valentine195/obsidian-initiative-tracker#20]
I like the ability to see the encounter difficulty, but it's a little too wordy for me. I think something like this might be a nice way to do it:

![Obsidian_U3sWZsxK1H](https://user-images.githubusercontent.com/54087190/145762021-ae3542d2-2248-41e9-8f3f-e6366d45e064.png) ![ygK3apI965](https://user-images.githubusercontent.com/54087190/145762094-948d065c-cab9-40e2-9e5d-1282fff47204.png)

Also maybe taking the colors from the bar to color the words would also be pretty neat. Then the hover tooltip can keep the full wording while keeping everything else nice and compact.

**Comment by valentine195** [comment]
![image](https://user-images.githubusercontent.com/38669521/145860196-01fa0ae6-9dff-4f8a-8d81-4453e773d978.png)

thoughts

**Comment by SlRvb** [comment]
Yep, that's exactly what I wanted 🥳

**Comment by valentine195** [comment]
Implemented with v5+
